parser.py

```diff
@@ -598,6 +606,7 @@ def main():
     data_folder = os.path.join(input_folder, 'data')
     account_js_filename = os.path.join(data_folder, 'account.js')
     log_path = os.path.join(output_media_folder_name, 'download_log.txt')
+    state_path = 'download_state.json'
```

Comment:
Move to PathConfig and use `cache` if we agree on that.

Reply:
FYI: the class `PathConfig` has been merged into `main` today via PR #115, but it does not have a path for `cache` yet.

I have just now published PR draft #120 which shall solve #99, and it introduces `PathConfig.dir_output_cache`, which is only used for a single file there.

Reply:
PR #120 is merged now, so you can use `state_path = os.path.join(paths.dir_output_cache, 'download_state.json')`